### ISSUE TITLE
Add decimal precession in toHuman

### ISSUE THREAD

**Issue by willemneal** [near/units-js#2]
```
near-units -h 1113144813545368235193 yN 
```
returns `1.113144813545368235193 mN`

which is not very human readable.

# Possible solution

Add argument to `toHuman` for how many decimal points, perhaps with a default of 2?

```
near-units -h 4 1113144813545368235193 yN 
```
returns `1.1131 mN`

**Comment by ryancwalsh** [comment]
+1

**Comment by ryancwalsh** [comment]
Currently a column of values like this isn't easily comparable at a glance (but I would like it to be, and I'll make it right-aligned with monospace font):

![image](https://user-images.githubusercontent.com/2086493/191600584-bfb26aa8-3de6-4353-8a5a-1860a52e880b.png)

**Comment by willemneal** [comment]
Yeah you should make another issue about adding an option to force a unit.
